framework/midi/internal/sequencer.cpp

```diff
@@ -363,15 +346,15 @@ bool Sequencer::sendChanEvents(const Channel& chan, uint32_t ticks)
             return ret;
         }
 
-        if (state.muted || event.type == MIDI_EOT || event.type == META_TEMPO) {
+        if (!m_isPlayTickSet) {
+            m_playTick = event.tick;
+            m_isPlayTickSet = true;
+        }
+
+        if (state.muted || event.type == MIDI_EOT || event.type == META_TEMPO || event.type == ME_TICK1 || event.type == ME_TICK2) {

```

Comment:
Better to use a set of types here, something like that:

QSet allowedTypes {  ... }

if (!state.muted && allowedTypes.contains(type)) { ... }

Reply:
I also thought about this and decided that if at least one more type is added (and ceases to fit on one line), then I will change it into a set